15/umbraco-commerce/how-to-guides/configuring-cart-cleanup.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Learn how to configure a cart cleanup routine.
+---
+
+# Configuring Cart Cleanup
+
+{% hint style="info" %}
+Available from Umbraco Commerce 15.1.0
+{% endhint %}
+
+By default Umbraco Commerce will keep all created carts indefinately. Over time this can obviously become an issue. To assist with this is it possible to configure a cart cleanup routine to delete carts older than a pre-configured time interval.
+
+This service can be enabled and configured in the `appSettings.json`
+
+```json
+{
+    "Umbraco" : {
+        "Commerce": {
+            "CartCleanupPolicy": {
+                "EnableCleanup": true,
+                "KeepCartsForDays": 800,
+                // Below settings are optional
+                "FirstRunTime": string; the time to first run the scheduled cleanup task, in crontab format
+                "Period": string; how often to run the task, in timespan format
+                // Configure diffrent policies per store

```

Comment:
Instead of explaining the configuration keys in the JSON code, it would be better to add a table or a list where they are defined.
We could put the list/table below the code snippet.
This will also make it more consistent with how we handle docs for other configuration files.

Reply:
Updated 👍